backend/user_management/routers/brand.py

```
@@ -58,19 +57,21 @@ async def get_my_brand(db: Session = Depends(get_db), current_user: models.User
     return brand
 
 
-
-@router.put("/brands/me", response_model=schemas.BrandOut)
-async def update_brand(brand: schemas.BrandCreate, db: Session = Depends(get_db), current_user: models.User = Depends(get_current_user)):
-    
-
+@router.patch("/brands/me", response_model=schemas.BrandOut)
+async def update_brand(
```

Ruff (0.8.2)

Missing return type annotation for public function `update_brand`

(ANN201)